opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -2454,15 +2517,6 @@ namespace detail {
 #endif
         }
 
-        // compute PAV and PORV for every regions.
```

Comment:
Thanks @GitPaean and @blattms for spotting this parallel bug and suggesting the fix.

@blattms: since we now want to compute the weighted pressure (```sd_.fip[SimulatorData::FIP_WEIGHTED_PRESSURE]```) for every cell, do you have a suggestion on the most efficient way to achieve this (wrt data communication vs synchronization)? Essentially, we first need to compute the sum of the hydrocarbon pore volume for each region, and then divide each cell (in the respectrive regions) by this.